nlp4all/models.py

```diff
@@ -322,18 +322,30 @@ class User(db.Model, UserMixin):
 
     def get_reset_token(self, expires_sec=1800):
         """Get a reset token."""
-        jws = Serializer(app.config["SECRET_KEY"], expires_sec)
-        return jws.dumps({"user_id": self.id}).decode("utf-8")
+        reset_token = jwt.encode(
+            {
+                "user_id": self.id,
+                "exp": datetime.now(tz=timezone.utc)
+                        + timedelta(seconds=expires_sec)
+            },
+            app.config['SECRET_KEY'],
+            algorithm="HS256"
+        )
+        return reset_token
 
     @staticmethod
     def verify_reset_token(token):
```

Comment:
one other thing, the docstring has gone which makes linting fail, you could add something like

```python
"""Generate a password reset token

        Parameters:
            expires_sec (int): Number of seconds the token remains valid

        Returns:
            reset_token (str): The password reset token
"""
```

also as you've touched this function, feel free to add type hinting

e.g.

```python
def get_reset_token(self, expires_sec: int = 1800) -> str:
```

Reply:
This should be done now :)